### PR TITLE
Make Security Dripped Out Again

### PR DESCRIPTION
**Pull request by Tmanzxd**
Leadup to PR #656. The Loadout fix for #145. Adds and makes a lot of loadout options for Security. Full Change Log Below:

**Change Notes:**

- Changed Wizden Security Vest sprites to DeltaV Sprites.

- Removed Wizden Security Vests from loadouts (can be re-added later in another PR if sentiment changes.)

- Added Plate Carriers and Durathread Vests to loadouts

- Made non-specialist Security apparel generic to Security. Charge Batong with style now!

- Added Flashes to Security Loadouts.

- Added Security Armor Vests to Security Loadouts.

- Added Detective Coats and Vests to Security Loadouts.

- Added Warden Armored Jacket to Warden Loadouts.

- Added Head of Security's Trenchcoat to Loadouts.

- Added Security gear & accessories to Loadouts.

- Added Brigmedic and Corpsman items for Corpsman!

- Added SecGlasses to Loadouts!

### CODE REVIEW

File: Resources/Prototypes/Loadouts/Jobs/security.yml [hunk 1]
```diff
@@ -352,33 +409,233 @@
...
+  id: LoadoutClothingBeltCorpsmanWebbing
+  category: Jobs
+  cost: 1
+  exclusive: true
+  requirements:
+    - !type:CharacterJobRequirement
+      jobs:
+        - Brigmedic
+  items:
+    - ClothingBeltCorpsmanWebbing
```
Comment (VMSolidus) [inline Resources/Prototypes/Loadouts/Jobs/security.yml:429]:
```suggestion
  items:
    - ClothingBeltCorpsmanWebbingFilled
```

**Review by VMSolidus — APPROVED** [review]
Looks fine other than screwing the Corpsman out of his medical supplies if he buys this item. Also, this item doesn't actually need to be here anyway because Corpsman already spawns with the Filled Webbing, making this a trap option.

image

**Comment by DangerRevolution** [comment]
BrigMedic backpack test fail.

**Comment by Tmanzxd** [comment]
Guh I can check it out once I clock off work

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml [hunk 2]
```diff
@@ -21,8 +21,7 @@
   components:
   - type: StorageFill
     contents:
-      - id: Flash
-#      - id: MagazinePistol # DeltaV - Security doesn't get an extra mag
+      - id: MagazinePistol # EinsteinEngines - Security does get an extra mag
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml:24]:
```suggestion
      - id: MagazinePistol
```

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml [hunk 3]
```diff
@@ -19,8 +19,7 @@
   components:
   - type: StorageFill
     contents:
-      - id: Flash
-#      - id: MagazinePistol # DeltaV - Security doesn't get an extra mag
+      - id: MagazinePistol # EinsteinEngines - Security does get an extra mag
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml:22]:
```suggestion
      - id: MagazinePistol
```

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/satchel.yml [hunk 4]
```diff
@@ -33,8 +33,7 @@
   components:
   - type: StorageFill
     contents:
-      - id: Flash
-#      - id: MagazinePistol # DeltaV - Security doesn't get an extra mag
+      - id: MagazinePistol # EinsteinEngines - Security does get an extra mag
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/satchel.yml:36]:
```suggestion
      - id: MagazinePistol
```

File: Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/backpacks.yml [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
     contents:
       - id: EmergencyRollerBedSpawnFolded
       - id: BodyBagFolded
       - id: Portafib
-    #  - id: Flash
       - id: BruteAutoInjector
         amount: 2
       - id: BurnAutoInjector
         amount: 2
+      - id: MagazinePistol # EinsteinEngines - Security does get an extra mag
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/backpacks.yml:15]:
```suggestion
      - id: MagazinePistol
```

File: Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/backpacks.yml
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/backpacks.yml:1]: Move this backpack out of the DeltaV folder

File: Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/duffelbag.yml
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/duffelbag.yml:1]: Move this backpack out of the DeltaV folder

File: Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/satchel.yml
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/satchel.yml:1]: Move this backpack out of the DeltaV folder

File: Resources/Prototypes/DeltaV/Roles/Jobs/Security/brigmedic.yml [hunk 6]
```diff
@@ -32,7 +32,7 @@
   equipment:
     jumpsuit: ClothingUniformJumpsuitBrigmedic
     outerClothing: ClothingOuterArmorPlateCarrier
-    back: ClothingBackpackBrigmedicDeltaVFilled
+    back: ClothingBackpackBrigmedicFilled #This will need fixed later. I will do that when I change the starting equipment in favor of the loadout system.
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Roles/Jobs/Security/brigmedic.yml:35]: Fix this before merge

File: Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml:1]: These sprites are genuinely look like shit and are completely obsolete on EE. Do not put these inside loadouts.

Comment (DangerRevolution) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml:1]: Need to check if these are Wizden sprites... will check the PR.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml [hunk 7]
```diff
@@ -40,6 +40,49 @@
...
+    sprite: Clothing/OuterClothing/Vests/security_slim.rsi
+  - type: Armor #Based on /tg/ but slightly compensated to fit the fact that armor stacks in SS14.
+    modifiers:
+      coefficients:
+        Blunt: 0.70
+        Slash: 0.70
+        Piercing: 0.70 #Can save you, but bullets will still hurt. Will take about 10 shots from a Viper before critting, as opposed to 7 while unarmored and 16~ with a bulletproof vest.
+        Heat: 0.80
+  - type: ExplosionResistance
+    damageCoefficient: 0.90
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml:84]: Why are these identical except for sprite? Parent one off the other and leave it be.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml [hunk 8]
```diff
@@ -417,13 +418,15 @@
     sprite: Clothing/OuterClothing/WinterCoats/coatsec.rsi
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHatHoodWinterSec
-  - type: Armor # DeltaV - adds resists to coat
+  - type: Armor # EinsteinEngines - makes equivalent to vests
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml:421]:
```suggestion
  - type: Armor

```

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

**Comment by DangerRevolution** [comment]
balance changes like this really shouldn't be in a loadouts PR

File: Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml
Comment (DangerRevolution) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml:1]: Yeah, I don't like the Wizden sprites; don't use the obsolete vets and use the delta ones.

File: Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/duffelbag.yml
Comment (Tmanzxd) [reply Resources/Prototypes/DeltaV/Entities/Clothing/Back/backpacks.yml/duffelbag.yml:1]: Deleted the DeltaV file and moved it to Prototypes\Catalog\Fills\Backpacks\StarterGear\backpack.yml

File: Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml:1]: If you're removing these, you need to set up migrations files.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml
Comment (Tmanzxd) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml:1]: Will do

File: Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml
Comment (Tmanzxd) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml:1]: Put them into deltaMigrations.yml

Comment (DangerRevolution) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml:1]: > Put them into deltaMigrations.yml

eeMigrations.yml please

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml:1]: > > Put them into deltaMigrations.yml
> 
> 
> 
> eeMigrations.yml please

no, the normal one

Comment (DangerRevolution) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml:1]: > > > Put them into deltaMigrations.yml
> > 
> > 
> > eeMigrations.yml please
> 
> no, the normal one

no

**Comment by Tmanzxd** [comment]
Pain

File: Resources/Migrations/deltaMigrations.yml [hunk 9]
```diff
@@ -98,4 +98,4 @@ SpaceTickSpawnerNPC: SpaceTickSpawner
 
 # 2024-03-07
 LockableButtonBrig: LockableButtonSecurity
-SignDirectionalICU: SignDirectionalIcu
+SignDirectionalICU: SignDirectionalIcu
```
Comment (DangerRevolution) [inline Resources/Migrations/deltaMigrations.yml:101]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: Resources/Changelog/Changelog.yml [hunk 10]
```diff
@@ -4849,7 +4849,6 @@ Entries:
   changes:
     - type: Tweak
       message: Rouny Sprite Changed
-  id: 6207
```
Comment (DangerRevolution) [inline Resources/Changelog/Changelog.yml:4852]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml [hunk 11]
```diff
@@ -21,8 +21,55 @@
...
+      - id: BruteAutoInjector
+        amount: 2
+      - id: BurnAutoInjector
+        amount: 2
+      - id: MagazinePistol
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackDuffelBrigmedic
+  id: ClothingBackpackDuffelBrigmedicFilled
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml:45]: This needs to be in the duffel.yml file

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml [hunk 12]
```diff
@@ -21,8 +21,55 @@
...
+    #  - id: Flash
+      - id: BruteAutoInjector
+        amount: 2
+      - id: BurnAutoInjector
+        amount: 2
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackSatchelBrigmedic
+  id: ClothingBackpackSatchelBrigmedicFilled
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml:61]: This needs to be in the satchel.yml file

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
Okay.

Three things:

1. All resistance value changes need to be in a seperate PR.
2. Wizden vests are to be left alone, I despise the sprites. The dynamic between plate carrier and durathread vest is also more interesting. Just put the Delta vests into loadouts.
3. Remove the armour --> vests change you did, leave that for the "seperate" PR.

**Comment by Tmanzxd** [comment]
Will port over durathread vests from DeltaV once I get back from work.

**Comment by Tmanzxd** [comment]
Accidentally deleted vests.rsi, porting over the durathread vest now

File: Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml [hunk 13]
```diff
@@ -417,12 +418,13 @@
     sprite: Clothing/OuterClothing/WinterCoats/coatsec.rsi
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHatHoodWinterSec
-  - type: Armor # DeltaV - adds resists to coat
+  - type: Armor
+
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml:422]:
```suggestion
```

File: Resources/Prototypes/Loadouts/items.yml [hunk 14]
```diff
@@ -469,6 +469,21 @@
...
+  items:
+    - Flash
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Security
+    - !type:CharacterJobRequirement
+       inverted: true
+       jobs:
+         - Brigmedic
```
Comment (DangerRevolution) [inline Resources/Prototypes/Loadouts/items.yml:485]: Why?

File: Resources/Textures/Clothing/OuterClothing/Armor/vest.rsi/meta.json [hunk 15]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (DangerRevolution) [inline Resources/Textures/Clothing/OuterClothing/Armor/vest.rsi/meta.json:1]: What's this?

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml [hunk 16]
```diff
@@ -34,6 +33,15 @@
     sprite: Clothing/OuterClothing/Armor/security_slim.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Armor/security_slim.rsi
+  - type: Armor #Based on /tg/ but slightly compensated to fit the fact that armor stacks in SS14.
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml:36]: Remove this entire part, don't touch the Wizden vest.

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
This seems to be missing most of the items from the description being added into loadouts

File: Resources/Textures/Clothing/OuterClothing/Armor/security_slim.rsi/equipped-OUTERCLOTHING.png
Comment (DangerRevolution) [inline Resources/Textures/Clothing/OuterClothing/Armor/security_slim.rsi/equipped-OUTERCLOTHING.png:1]: Undo this too, I like *only* the Delta-V vest sprite.

File: Resources/Textures/Clothing/OuterClothing/Armor/security_slim.rsi/icon.png
Comment (DangerRevolution) [inline Resources/Textures/Clothing/OuterClothing/Armor/security_slim.rsi/icon.png:1]: Undo this too, I like *only* the Delta-V vest sprite.

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml (hunk 16 above)
Comment (Tmanzxd) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml:36]: Commented out the modifiers but not the sprites to ensure that compiling runs smoohthly.

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/satchel.yml [hunk 17]
```diff
@@ -28,33 +28,38 @@
...
+  id: ClothingBackpackSatchelBrigmedicFilled
   components:
   - type: StorageFill
     contents:
-      - id: Flash
-#      - id: MagazinePistol # DeltaV - Security doesn't get an extra mag
+      - id: EmergencyRollerBedSpawnFolded
+      - id: BodyBagFolded
+      - id: Portafib
+    #  - id: Flash
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/satchel.yml:39]:
```suggestion
```

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

**Review by DangerRevolution — APPROVED** [review]
merge when tests pass

**Pull request merged**